internal/usage/usage.go

```diff
@@ -377,7 +377,7 @@ dbName and collection are required only for built-in roles.`
 	EnableRealtimePerformancePanel            = "Flag that enables the Real Time Performance Panel project setting."
 	DisableRealtimePerformancePanel           = "Flag that disables the Real Time Performance Panel project setting."
 	CloudProvider                             = "Name of the provider of the cloud service where Atlas can access the S3 bucket. Atlas only supports AWS."
-	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket."
+	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving a cloud provider access role, see: https://dochub.mongodb.org/core/set-up-unified-aws-access/."
```

Comment:
Need to remove the trailing slash, I tried navigating to the URL and it doesn't work with the trailing slash
```suggestion
	ExportBucketIAMRoleID                     = "Unique identifier that Atlas assigns to the cloud provider access role for the bucket. To learn more about setting up and retrieving a cloud provider access role, see: https://dochub.mongodb.org/core/set-up-unified-aws-access."
```